k6-operator/Makefile

```diff
@@ -0,0 +1,23 @@
+.DEFAULT_GOAL := get-upstream
+
+CHART_VERSION=3.8.0
+APP_VERSION=0.0.16
+
+
+.PHONY: get-upstream
```

Comment:
In this repo, we usually call `upstream` to the files/manifests that we pull unchanged, so I don't like calling it `upstream` here, since the produced manifests have already been templated by our values

Reply:
what would be the preference? `gen-manifests` or something similar?